base-theme/layouts/partials/featured_course_cards.html

```diff
@@ -25,15 +25,13 @@ <h2 class="{{ $klass }}">Featured Courses</h2>
               {{- $url := delimit (slice (strings.TrimSuffix "/" $staticApiBaseUrl) $urlPath "/data.json") "" -}}
               {{ with resources.GetRemote $url }}
                 {{ with .Err }}
-                  {{ $errorMessage := printf "Featured courses carousel on %v failed to fetch %v with error: %v" site.BaseURL $url . }}
-                  {{ partial "sentry_capture_message.html" $errorMessage }}
+                  {{ errorf "Featured courses carousel on %v failed to fetch %v with error: %v" site.BaseURL $url . }}
                 {{ else }}
                   {{- $courseData := . | unmarshal -}}
                   {{ partial "course_carousel_card.html" (dict "itemsInCarousel" $itemsInCarousel "courseData" $courseData "index" $index "urlPath" $urlPath "numCourses" (len $courses))}}
                 {{ end }}
               {{ else }}
-                {{ $errorMessage := printf "Failed to fetch featured courses through %v on %v" $url site.BaseURL }}
-                {{ partial "sentry_capture_message.html" $errorMessage }}
+                {{ errorf "Failed to fetch featured courses through %v on %v" $url site.BaseURL }}
```

Comment:
Adding this comment here, but it should be considered in all the other places we're now throwing errors. I realize that `site.BaseURL` was used previously here, but I'm not sure it's the best thing to use. There are a couple situations in which this would be inaccurate:

 - In local development if you set `STATIC_API_BASE_URL` to RC or production, you will get a message like `Failed to fetch new course card through https://ocw.mit.edu/courses/6-4210-robotic-manipulation-fall-2022/data.jsonmeow on //localhost:3000/`. This is less of an issue and honestly somewhat expected because you're overriding the value for local dev.
 - In production, currently we set `site.BaseURL` to `/courses/course_id`, an absolute but not fully qualified URL. In this case the error would be more accurate, but the extra information isn't really helpful as the whole URL is already displayed with `$url`.
 - If we go with my proposed solution in https://github.com/mitodl/ocw-hugo-themes/issues/1013, we won't be using the `site.BaseURL` setting and all sites will just be built with relative URLs, with requests to shared static resources being rewritten to reference the root by Fastly.

Therefore, I think we should simply remove references to `site.BaseURL` from these error messages and simply display the URL from the request as that will be more accurate as to what request failed.